monai/apps/pathology/transforms/post/array.py

```diff
@@ -0,0 +1,336 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Optional, Sequence, Union
+
+import numpy as np
+import torch
+
+from monai.config.type_definitions import NdarrayOrTensor
+from monai.data.meta_tensor import get_track_meta
+from monai.transforms.croppad.array import BoundingRect
+
+from monai.transforms.post.array import GetInstanceLevelSegMap, Activations, AsDiscrete
+from monai.transforms.transform import Transform
+from monai.utils import convert_to_numpy, convert_to_tensor, optional_import
+from monai.utils.enums import TransformBackends
+
+find_contours, _ = optional_import("skimage.measure", name="find_contours")
+moments, _ = optional_import("skimage.measure", name="moments")
+
+
+__all__ = ["PostProcessHoVerNetOutput"]
+
+
+def _coords_to_pixel(current, previous) -> Sequence[int]:
```

Comment:
I know that the name of these functions come from the original script but they are not very descriptive so reduce the readability of the code. Can you improve their names if possible and update the docstring?

Reply:
Have updated it, thanks!